tls/s2n_psk.h

```diff
@@ -98,11 +98,16 @@ int s2n_psk_set_identity(struct s2n_psk *psk, const uint8_t *identity, uint16_t
 int s2n_psk_set_secret(struct s2n_psk *psk, const uint8_t *secret, uint16_t secret_size);
 int s2n_psk_set_hmac(struct s2n_psk *psk, s2n_psk_hmac hmac);
 
+int s2n_psk_get_type(struct s2n_psk *psk, uint8_t *type);
+int s2n_psk_get_identity_length(struct s2n_psk *psk, uint16_t *identity_length);
+int s2n_psk_get_identity(struct s2n_psk *psk, uint8_t *identity, uint16_t *identity_length);
+
 int s2n_connection_append_psk(struct s2n_connection *conn, struct s2n_psk *psk);
 
 typedef enum { S2N_PSK_MODE_RESUMPTION, S2N_PSK_MODE_EXTERNAL } s2n_psk_mode;
 int s2n_config_set_psk_mode(struct s2n_config *config, s2n_psk_mode mode);
 int s2n_connection_set_psk_mode(struct s2n_connection *conn, s2n_psk_mode mode);
+int s2n_connection_get_chosen_psk(struct s2n_connection *conn, struct s2n_psk *chosen_psk);
 
```

Comment:
We call this "chosen_psk" internally, but is there a better name for customers to understand? "active_psk"? "negotiated_psk"?

Reply:
Maybe we should make this an s2n_offered_psk?

Reply:
I think `negotiated_psk` psk might suit better than `active_psk` here.